### PR TITLE
Shorten names

### PR DESCRIPTION
**Pull request by cbrnr**
Some of the names are too verbose and could be shortened without losing their descriptiveness.

### CODE REVIEW

File: sleepecg/tests/test_heartbeat_detection.py [hunk 1]
```diff
@@ -1,43 +1,36 @@
 # Authors: Florian Hofer
 #
 # License: BSD (3-clause)
-
-"""Tests for heartbeat detection and detector evaluation."""
+"""Tests for heartbeat detection C extension."""
```
Comment (hofaflo) [inline sleepecg/tests/test_heartbeat_detection.py:4]:
```suggestion

"""Tests for heartbeat detection C extension."""
```

File: sleepecg/tests/test_heartbeats.py [hunk 2]
```diff
@@ -0,0 +1,43 @@
+# Authors: Florian Hofer
+#
+# License: BSD (3-clause)
+"""Tests for heartbeat detection and detector evaluation."""
```
Comment (hofaflo) [inline sleepecg/tests/test_heartbeats.py:4]:
```suggestion

"""Tests for heartbeat detection and detector evaluation."""
```

**Review by hofaflo — APPROVED** [review]
+1

**Comment by cbrnr** [comment]
@hofaflo ready to merge!

**Comment by hofaflo** [comment]
Thanks @cbrnr!

**Pull request merged**